src/components/skjema/innsatsgruppe/innsatsgruppe.tsx

```diff
@@ -12,35 +12,37 @@ export enum InnsatsgruppeType {
     VARIG_TILPASSET_INNSATS = 'VARIG_TILPASSET_INNSATS'
 }
 
+export const innsatsgrupper = [
```

Comment:
Alternativt så kan man ha en funksjon med switch/case som mapper enum til tekst. Da slipper man også å iterere over for å finne teksten etterpå.

Reply:
ja lurt